api/fsi.go

```diff
@@ -47,22 +48,30 @@ func (h *FSIHandlers) StatusHandler(routePrefix string) http.HandlerFunc {
 
 func (h *FSIHandlers) statusHandler(routePrefix string) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
+		useFSI := r.FormValue("fsi") == "true"
 		ref, err := DatasetRefFromPath(r.URL.Path[len(routePrefix):])
 		if err != nil {
 			util.WriteErrResponse(w, http.StatusBadRequest, fmt.Errorf("bad reference: %s", err.Error()))
 			return
 		}
 
 		res := []lib.StatusItem{}
-		if ref.Path != "" {
+		if !useFSI {
```

Comment:
let's switch the useFSI test to check for truth instead of falsy, call StatusForAlias and early return. Outdent the else clause & make StatusAtVersion the default.